.eslintrc.js

```diff
@@ -7,23 +7,34 @@ module.exports = {
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
+      rules: {
+        '@typescript-eslint/naming-convention': 0,
+        '@typescript-eslint/no-floating-promises': 1,
+        '@typescript-eslint/prefer-enum-initializers': 0,
+        '@typescript-eslint/prefer-readonly': 0,
```

Comment:
This one seems okay to enable since there are two and doesn't create breaking changes. What do you think?